### PR TITLE
Random fixes

### PR DESCRIPTION
**Pull request by jjjhill**
@ryanprice see changes to App.vue
see if that works for you for having a /dev/ folder and a /bsc/ folder for contracts, you'll have to uncomment that code and comment out the original imports 

other changes:
- fix sorting on editions on shop/crypt
- chain is detected on mount
- add icon for website

### CODE REVIEW

**Review by ryanprice — APPROVED** [review]
Verified chain changing state, works great
Sorting in shop and crypt look good, noticed the public folder
Looks good to me

**Review by ryanprice — COMMENTED** [review]
when there are changes in package.json, am I best to re-install ? i.e I do that manually correct ?

**Comment by mchigit** [comment]
> when there are changes in package.json, am I best to re-install ? i.e I do that manually correct ?

Yep. You can just run `yarn install`

File: src/components/layout/AppFooter.vue [hunk 1]
```diff
@@ -100,23 +100,23 @@ import {NETWORKS} from '../../util/constants/networks'
...
+          return 'eth-link';
+      }
     },
     isConnected() {
       return this.$store.state.web3.isConnected
     },
     network() {
-        //console.log("Chain id:",this.$store.state.web3.chainId);
-      const hexString = this.$store.state.web3.chainId
+      const hexString = `0x${this.$store.state.web3.chainId.toString(16)}`
```
Comment (mchigit) [inline src/components/layout/AppFooter.vue:117]: I thought chain ID is already in the format of `0x{smth}` from line 106? So the hex string is `0x0x{chainId}`?

**Review by mchigit — APPROVED** [review]
LGTM, just 1 question about chain ID

File: src/components/layout/AppFooter.vue (hunk 1 above)
Comment (jjjhill) [reply src/components/layout/AppFooter.vue:117]: i saved it as a `number` in the store, so its saved as 0x63 (just a base 16 number, not a string) for example, but if you to chainId.toString() on that it converts it to '99'. 0x63.toString(16) returns `63` and therefore gotta put a 0x in front of it

**Pull request merged**